src/frontend/components/ListingPageCard/ListingPagecard.js

```diff
@@ -1,17 +1,30 @@
 import React from "react";
 
+import { useNavigate } from "react-router-dom";
+import { useAuth, UseWishlist } from "../../contexts"
```

Comment:
doesn't affect the functionality but `useWishlist`* would've looked better wrt conventions.